migrations/versions/201509041515_f9bb2a03ceb_add_registration_tables.py

```diff
@@ -88,6 +100,7 @@ def upgrade():
 
 
 def downgrade():
+    op.drop_column('registration_form_items', 'current_data_id', schema='event_registration')
```

Comment:
We have a naming policy configured:

    'fk': 'fk_%(table_name)s_%(column_names)s_%(referred_table_name)s',

So you know the name of it ;)